design/resource-validation.md

```diff
@@ -121,22 +105,32 @@ NGF-specific validation will not include:
 
 - *All* validation done by CRDs. NGF will only repeat the validation that addresses (1) and (2) in the list above with
   extra rules required by NGINX but missing in the CRDs. For example, NGF will not ensure the limits of field values.
-- The validation done by the webhook (because it is done in the previous step).
+- The validation done by the webhook.
```

Comment:
we cannot leave this part without running the webhook validation in our control plane